plugins/module_utils/mso.py

```diff
@@ -159,6 +164,65 @@ def mso_hub_network_spec():
     )
 
 
+def format_message(err, resp):
+    return err
+
+
+def write_file(module, url, dest, content, resp):
```

Comment:
if this function was copied from another project we should add a comment that refer to it.

Reply:
okay